config/sidekiq.yml

```
@@ -1,8 +1,8 @@
 :queues:
   - default
-  - active_storage_transform
-  - active_storage_analysis
-  - active_storage_purge
+  - [active_storage_transform, 3]
+  - [active_storage_analysis, 1]
+  - [active_storage_purge, 1]
```

Attention, tu n'as pas besoin de mettre les priorités 
Je crois que ca vient d'une autre gem qui fait ca  : 

```suggestion
  - active_storage_transform
  - active_storage_analysis
  - active_storage_purge
```